### PR TITLE
Updates CONTRIBUTING.md architecture

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Adds function in gulp docs task to copy contributing doc.

## Removals

- Removes individual package CONTRIBUTING.md files.

## Changes

- Updates CONTRIBUTING doc copy.
- Makes documentation site use same contributing doc as the github repo.
- Merges documentation contributing doc and github repo doc.

## Testing

1. Ensure you have jekyll installed https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll
1. `cd docs && gulp docs && bundle exec jekyll serve`

## Notes

- I had to remove the "Guidance on how to contribute" heading because in the documentation site (where I'd think we care about the appearance more) it says 

```
Contributing
Guidance on how to contribute
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -151,7 +160,26 @@ features. For more information visit the [Babel documentation site]
...
+Ready to publish changes to npm?
+
+1. Ensure you're on `master` and `git pull` to confirm you're up-to-date.
+1. Export a personal access token called [`GITHUB_AUTH`](https://github.com/lerna/lerna-changelog#github-token).
+1. Run `yarn run changelog` and open `CHANGELOG.md` to see a preview
+of new changelog entries.
+We use a [tool](https://github.com/lerna/lerna-changelog#usage) that scans our
+Pull Requests for specific labels so if you see a PR missing from the changelog,
+ensure it has been labeled `breaking`, `enhancement`, `bug`,
+`documentation` or `internal`.
```
Comment (Scotchester) [inline CONTRIBUTING.md]: Trivial: Would be nice to add three spaces of indentation to the additional lines in this long list item.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
-# Guidance on how to contribute
-
 > All contributions to this project will be released under the CC0 public domain
 > dedication. By submitting a pull request or filing a bug, issue, or
 > feature request, you are agreeing to comply with this waiver of copyright interest.
-> Details can be found in our [TERMS](TERMS.md) and [LICENSE](LICENSE).
-
+> Details can be found in our [TERMS](https://github.com/cfpb/capital-framework/blob/master/TERMS.md) and [LICENSE](https://github.com/cfpb/capital-framework/blob/master/LICENSE).
```
Comment (Scotchester) [inline CONTRIBUTING.md]: We should stop abusing the `blockquote` formatting here. On the docs site, it just indents the paragraph, which looks odd. I'd say just let it be a regular paragraph and maybe put an `hr` underneath to separate the more practical section below.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,23 +14,28 @@ Use the issue tracker to suggest feature requests, report bugs, and ask question
...
 
-For example, if you wanted to change `cf-buttons`, here's what you'd do:
+For example, if you wanted to change `cf-buttons` and use it in
+[cfgov-refresh](https://github.com/cfpb/cfgov-refresh),
+here's what you'd do:
 
 1. `git clone git@github.com:cfpb/capital-framework.git`
 1. `cd capital-framework`
 1. `yarn` to install dependencies and set up [workspaces](https://yarnpkg.com/lang/en/docs/workspaces/)
+1. `git checkout master && git pull` to ensure you're on the latest changes.
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply CONTRIBUTING.md]: Made some edits here https://github.com/cfpb/capital-framework/pull/910/commits/6ba4a5c2b1cced56fc864bc9db4014867a4ba9ea

File: CONTRIBUTING.md (hunk 2 above)
Comment (anselmbradford) [reply CONTRIBUTING.md]: https://github.com/cfpb/capital-framework/pull/910/commits/010a88fe075bbee0e6e5c425d28f49551dc90744

File: CONTRIBUTING.md (hunk 1 above)
Comment (anselmbradford) [reply CONTRIBUTING.md]: https://github.com/cfpb/capital-framework/pull/910/commits/010a88fe075bbee0e6e5c425d28f49551dc90744

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -54,25 +68,20 @@ Automated tests can be run with the command `yarn test`.
...
-- Make sure you're on the gh-pages branch
-- [Run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
-- Run `yarn run cf-link`
-- Run `yarn start`
-- Open a browser at http://localhost:3000/ and navigate to the component you've changed
-- When you've finished testing run `yarn run cf-unlink` to reset your components to the published releases
+If you're hacking on a component and want to test it in the documentation site
+follow the following steps.
 
+1. [Set up Jekyll to run the docs site locally](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll),
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply CONTRIBUTING.md]: https://github.com/cfpb/capital-framework/pull/910/commits/010a88fe075bbee0e6e5c425d28f49551dc90744

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -71,12 +73,14 @@ Automated tests can be run with the command `yarn test`.
...
-1. [Set up Jekyll to run the docs site locally](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll),
-if it's not already set up.
+1. Ensure that you have the Bundler Ruby gem installed
+   by running `gem install bundler`.
+   ([more info](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll#requirements)).
 1. Check out a working branch.
 1. Run `gulp docs`.
 1. Go into the documentation directory with `cd docs`.
-1. Run `bundle exec jekyll serve` and visit http://127.0.0.1:4000.
+1. Run `bundle install` if you haven't before.
```
Comment (Scotchester) [inline CONTRIBUTING.md]: Still missing a mention of running `bundle update` periodically after the initial install.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -39,7 +40,7 @@
 - [ ] New functions are documented (with a description, list of inputs, and expected output)
 - [ ] Placeholder code is flagged / future todos are captured in comments
 - [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
-- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
+- [ ] Project documentation has been updated with `gulp docs` command
```
Comment (Scotchester) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Project documentation has been updated in this PR with the `gulp docs` command
  (we no longer use the gh-pages branch for docs, so doc updates should happen in the same PR as CF source code updates)
```

Just to be extra clear.

File: CONTRIBUTING.md (hunk 5 above)
Comment (anselmbradford) [reply CONTRIBUTING.md]: ARGH had this addition unsaved in a tab :(

Comment (anselmbradford) [reply CONTRIBUTING.md]: https://github.com/cfpb/capital-framework/pull/910/commits/5efd96cb27e052cea18fecc726f07932167b7b6f

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (anselmbradford) [reply .github/PULL_REQUEST_TEMPLATE.md]: https://github.com/cfpb/capital-framework/pull/910/commits/862df9b4c9ba95dc33564c7f297060709967eedd

Comment (anselmbradford) [reply .github/PULL_REQUEST_TEMPLATE.md]: https://github.com/cfpb/capital-framework/pull/910/commits/862df9b4c9ba95dc33564c7f297060709967eedd

**Pull request merged**